dcs/urls.py

```diff
@@ -9,7 +9,7 @@
 
     url(r'^$', 'dcs.views.home', name='home'), # mock, CHANGE later!
     url(r'^admin/', include(admin.site.urls)),
-    url(r'^multimedia/', include('multimedia.urls')),
+    url(r'^multimedia/', include('multimedia.urls', namespace='multimedia')),
```

Comment:
@digawp multimedia is just a default URL I wrote since it is under multimedia.
What u said is correct, we should not use `/multimedia` in our url routing because its ugly.

The better urls are something like:
```
/books/
/movies/
/musics/
/albums/
/apps/
```